docker/types/image.py

```diff
@@ -0,0 +1,35 @@
+from .base import DictType
+
+
+class Platform(DictType):
+    def __init__(self, **kwargs):
+        architecture = kwargs.get('architecture')
+        os = kwargs.get('os')
+
+        if architecture is None and os is None:
+            raise ValueError("At least one of 'architecture' or 'os' must be provided")
```

Comment:
This message doesn't match the logic. you are requiring both and not just one.  I'd either correct the logic or correct the statement.